### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'pluginOptions')

### DISCUSSION THREAD

**Discussion by muhammad-grayphite** [richardeschloss/nuxt-highcharts#59] [Show and tell]
"nuxt": "^2.15.0"
    "nuxt-highcharts": "^2.0.4",

t.$config.nuxtHighcharts is undefined in the script. @richardeschloss plz look into it. I have attached the screen shot as well.
please somebody help me if there is any workaround.

here is my code in nuxt config file.
```
{
  modules: [
    // Simple usage
    'nuxt-highcharts',
  ],
  highcharts: {
    /* module options */
  }
}
```

<img width="1266" alt="Screenshot 2023-01-24 at 10 28 19 PM" src="https://user-images.githubusercontent.com/85872908/214366196-861798ab-c751-4e44-8beb-98d1e56af035.png">
<img width="782" alt="Screenshot 2023-01-24 at 10 34 53 PM" src="https://user-images.githubusercontent.com/85872908/214366345-9acf303a-870f-4d4f-a4f0-fef64c8ca6e1.png">

**Comment by richardeschloss** [comment]
It's been so long since I worked on Nuxt2 but it's important for `publicRuntimeConfig` to defined.
https://github.com/richardeschloss/nuxt-highcharts/blob/60a0a8fb8dc5c23a65e4c52f6a76c4acb9a38bef/lib/module.js#L30

That's how it gets wired up to `this.$config`.

**Reply by muhammad-grayphite** [reply]
I have defined `publicRuntimeConfig` in nuxt config but I'm still getting this issue.
Interesting thing is its only happen one time at first time we load the site on browser and we generating the static site and deploying it on the netlify. maybe this helps.

**Reply by richardeschloss** [reply]
FWIW, this was the working [nuxt.config](https://github.com/richardeschloss/nuxt-highcharts/blob/60a0a8fb8dc5c23a65e4c52f6a76c4acb9a38bef/nuxt.config.js) for the Nuxt2 version, which has been successfully deployed to Netlify. I'm not sure why you're encountering issues, but perhaps you may want to specify `target` to "static".
